homeassistant/components/otbr/config_flow.py

```diff
@@ -53,6 +57,18 @@ async def async_step_hassio(self, discovery_info: HassioServiceInfo) -> FlowResu
 
         config = discovery_info.config
         url = f"http://{config['host']}:{config['port']}"
+
+        api = python_otbr_api.OTBR(url, async_get_clientsession(self.hass), 10)
+        try:
+            if await api.get_active_dataset_tlvs() is None:
+                await api.async_create_active_dataset(
```

Comment:
We should first ask the Thread integration if it has a preferred one. Do you want to do that in a follow-up PR?